### PR TITLE
Add darkfield correction option to electron count job

### LINKED ISSUE #502: add row-dark option
We should add an option to the "Count job" parameter window and script to use normal, row-dark mean, row-dark median.
image

### PR DESCRIPTION
**Pull request by alesgenova**
Fixed #502 

Also, depends on changes to the `electron_count_cory.py` script

### CODE REVIEW

**Comment by swelborn** [comment]
hey @alesgenova, LGTM other than that minor wraparound text thing. we will add the template params for streaming later.

**Comment by alesgenova** [comment]
> hey @alesgenova, LGTM other than that minor wraparound text thing. we will add the template params for streaming later.

I'm not sure I know what the wraparound text thing is :) Can you elaborate so I can fix it?

**Comment by swelborn** [comment]
@alesgenova sorry for my delay - can you see the images in my comments in the review?

**Comment by alesgenova** [comment]
@swelborn No, I can't see any comments either

File: frontend/distiller/src/components/streaming-dialog.tsx [hunk 1]
```diff
@@ -111,39 +124,57 @@ const StreamingDialog: React.FC<Props> = (props) => {
...
+          <TimeField
+            label="Session time (HH:MM)"
+            value={time}
+            onChange={(newTime) =>
+              newTime ? setTime(newTime) : setTime(defaultTime)
+            }
+            format="HH:mm"
+            minTime={minTime}
+            maxTime={maxTime}
+            formatDensity="spacious"
```
[the remark this answers is not in the document]
Comment (alesgenova) [reply frontend/distiller/src/components/streaming-dialog.tsx:172]: Fixed it

**Comment by swelborn** [comment]
updated to latest faust jobs image and tested streaming on this branch. it works as expected.

Screenshot 2024-02-05 at 4 17 54 PM

**Review by cjh1 — APPROVED** [review]
LGTM

**Pull request merged**